src/utilities/utility.ts

```diff
@@ -105,6 +105,24 @@ export async function getWorkflowFilePath(githubToken: string): Promise<string>
     return Promise.resolve(workflowFilePath);
 }
 
+export async function getRepositoryId(githubToken: string): Promise<number> {
+    const githubClient = new GitHubClient(process.env.GITHUB_REPOSITORY, githubToken);
+    let repoId = 0;
+    const response = await githubClient.getRepo();
+    if (response) {
+        if (response.statusCode == StatusCodes.OK
+            && response.body) {

```

Comment:
Don't you need to handle the case where `response.body` is null and StatusCode is OK.

Reply:
Will do.